Sources/AirRobeWidget/Service/Endpoints/AirRobeWidgetEndpoints.swift

```diff
@@ -58,4 +58,20 @@ extension AirRobeEndpoint {
         )
     }
 
+    static func identifyOrder(orderId: String, orderOptedIn: Bool) -> AirRobeEndpoint {
```

Comment:
@michaeldawson 
Please check this whether it's what was expected.

Reply:
LGTM